desktop/src/main/java/bisq/desktop/components/TextFieldWithCopyIcon.java

```diff
@@ -39,7 +39,7 @@
     private final StringProperty text = new SimpleStringProperty();
     private final TextField textField;
     private boolean copyWithoutCurrencyPostFix;
-
+    private boolean copyWithoutBeforeSlash;
```

Comment:
As this code is independent of `/` I think it would be better to name it to `copyTextAfterDelimiter`